BCStudentSoftwareDevTeam/celts
# Updated sidebar display to show unapproved courses and interested students

## Issue
Fixes issue #1146 and issue #1147 

We were wanting to the admin to be able to see important information like courses they need to approve and students who expressed interest since those require their action. 

## Changes
- Created a new controller in `admin/routes.py` called getSidebarInformation that gets all of the unapproved courses and all of the students who expressed interest in the CCE minor to display in the sidebar.
- Created a new JavaScript file `sidebar.js` that calls an ajax request on sidebar load and adds the numbers to the sidebar if applicable.
- Added hovers to each of the sidebar components that now have numbers next to them to display what the numbers mean.
- Modified `sidebar.html` to include the new script and gave the Course Management and Minor Management tabs an id to be referenced by the JavaScript script.

## Testing
- switch to branch `1146-interested-students-sidebar`
- use ramsayb2 and open up the admin dropdown.
- Hover over Course Management and Minor Management to make sure a popover appears and describes what the numbers mean.
- verify that Course Management has (1), that interested students has (4), and the Admin dropdown has (5).
- make a student express interest and a new course for the current term and make sure the numbers update when you return to ramsayb2.
- look over code.
